variables.tf

```diff
@@ -21,79 +21,108 @@ variable "cluster_identity_oidc_issuer_arn" {
 
 variable "helm_chart_name" {
   type        = string
-  default     = "vector-agent"
+  default     = "vector"
   description = "Helm chart name to be installed"
 }
 
 variable "helm_chart_version" {
   type        = string
-  default     = "0.15.1"
+  default     = "0.13.0"
   description = "Version of the Helm chart"
 }
 
 variable "helm_release_name" {
   type        = string
-  default     = "vector-agent"
+  default     = "vector"
   description = "Helm release name"
 }
 
 variable "helm_repo_url" {
   type        = string
-  default     = "https://packages.timber.io/helm/latest"
+  default     = "https://helm.vector.dev"
   description = "Helm repository"
 }
 
 variable "helm_create_namespace" {
   type        = bool
   default     = true
-  description = "Whether to create k8s namespace with name defined by `k8s_namespace`"
+  description = "Whether to create k8s namespace with name defined by `namespace`"
 }
 
-variable "k8s_namespace" {
+variable "namespace" {
   type        = string
   default     = "kube-system"
   description = "The K8s namespace in which the vector agent will be installed"
 }
 
-variable "k8s_rbac_create" {
+variable "rbac_create" {
   type        = bool
   default     = true
   description = "Whether to create and use RBAC resources"
 }
 
-variable "k8s_service_account_create" {
+variable "service_account_create" {
   type        = bool
   default     = true
   description = "Whether to create Service Account"
 }
 
-variable "k8s_irsa_role_create" {
+variable "service_account_name" {
+  default     = "aws-vector-agent-log"
+  description = "The k8s aws-vector-agent-log service account name"
+}
+
+variable "irsa_role_create" {
   type        = bool
   default     = true
   description = "Whether to create IRSA role and annotate service account"
 }
 
-variable "k8s_irsa_role_name_prefix" {
+variable "irsa_role_name_prefix" {
   type        = string
-  default     = "eks-vector-irsa"
+  default     = "eks-irsa"
   description = "The IRSA role name prefix for vector"
 }
 
-variable "k8s_assume_role_arn" {
+variable "irsa_tags" {
+  type        = map(string)
+  default     = {}
+  description = "IRSA resources tags"
+}
+
+variable "irsa_assume_role_enabled" {
+  type        = bool
+  default     = false
+  description = "Whether IRSA is allowed to assume role defined by assume_role_arn. Useful for hosted zones in another AWS account."
```

Comment:
second part of description doesn't make sense: "Useful for hosted zones in another AWS account." Proprably copy/paste typo ?

Reply:
yes, copy/paste typo